.changeset/twenty-spies-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@caravan/psbt": major
```

Comment:
I think we can make this a patch. It's not backwards incompatible in any way and isn't a new feature. Just an (important) bug fix.